monitor_other_boards.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+from datetime import datetime
+import subprocess
+
+import slack_reporter
+
+#               SSH target               name
+all_boards = (("odroid@odroid",         "odroid-C4"),
+              ("orangepi@oliviaorange", "orange pi 02"),
+              ("viam@viam",             "Up 4000"),
+              ("viam@orinnanodevkit2",  "Orin Nano"),
+              # The Orin AGX is unable to get on tailscale, and mDNS is not
+              # very reliable. but its IP address (almost?) never changes.
+              ("viam@10.1.8.37",        "Orin AGX"),
```

Comment:
This line might be trouble if we execute this script from a cloud provider, rather than something in the office. @abe-winter, any advice?

Reply:
Thanks to Abe for figuring out why TailScale wasn't starting up (we had a broken file in `/etc/apt/sources.list.d/`). I've put the board on the VPN, and this line should be way better now.